src/actions/serve.js

```diff
@@ -36,6 +36,7 @@ module.exports = () => {
         process.env.LNG_SERVE_CACHE_TIME ? '-c' + process.env.LNG_SERVE_CACHE_TIME : '-c-1',
         process.env.LNG_SERVE_PORT ? '-p' + process.env.LNG_SERVE_PORT : false,
         process.env.LNG_SERVE_PROXY ? '-P' + process.env.LNG_SERVE_PROXY : false,
+        process.env.LNG_SERVE_CORS ? '--cors=' + (process.env.LNG_SERVE_CORS === 'true' ? '*' : process.env.LNG_SERVE_CORS) : false,
```

Comment:
If set to "false" this will produce the argument:
```
--cors=false
```

I'd opt for not checking for "true" or "false" and just advising the user in the documentation to set the value of the environment variable to whatever they'd like the CORS allow origin to be. We should also wrap/escape the string to prevent the environment variable from leaking into other parameters.

Reply:
fixed

Reply:
https://github.com/rdkcentral/Lightning-CLI/pull/225/files#diff-b2e392bd439cb20d00dc506078a38402143d62705791693f89bd7e151b3a94a5R33